pygeos/test/test_strtree.py

```diff
@@ -0,0 +1,72 @@
+import pygeos
+from pygeos import box
+import pytest
+import numpy as np
+from numpy.testing import assert_array_equal
+from .common import point, empty, assert_increases_refcount, assert_decreases_refcount
+
+
+@pytest.fixture
+def tree():
+    geoms = pygeos.points(np.arange(10), np.arange(10))
+    yield pygeos.STRtree(geoms)
+
+
+def test_init_with_none():
+    pygeos.STRtree(np.array([None]))
```

Comment:
Maybe also do an actual query here? (since the tree is actually only created on first query, that seems relevant for the all-None case that also this query doesn't segfault or so)